scripts/getMetadataThumbnail.cdc

```diff
@@ -0,0 +1,9 @@
+import LoveSpreads from "../LoveSpreads.cdc"
+
+pub fun main(account: Address, id: UInt64,): String {
+    let ownerCollection = getAccount(account).getCapability(LoveSpreads.CollectionPublicPath)
+                                    .borrow<&LoveSpreads.Collection{LoveSpreads.CollectionPublic}>()
+                                    ?? panic("This person does not have a LoveSpreads Collection set up properly.")
+    let nftRef = ownerCollection.borrowAuthNFT(id: id) ?? panic("This account does not own an NFT with this id.")
```

Comment:
This works for a test, but in case you'll have to do such stuff again I think that you don't need the most specified type here (you get a LoveSpreads NFT with borrowAuthNFT), using ViewResolver interface is enough (that's how marketplaces work, they don't know about particular NFT types, but they use common views).